README.md

```diff
@@ -76,11 +69,9 @@ Optionally [VSCode](https://code.visualstudio.com/) with extension `ESLint`
 
 * Download a database GUI like [DBeaver](https://dbeaver.io/download/) and connect to the local MySQL server
 
-  Create database by executing the script in `DBeaver`
+  Create database by executing the following in `DBeaver`:
   ```
-  CREATE DATABASE stack_overflow_v2;
-  USE stack_overflow_v2;
-  SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
+  CREATE DATABASE askgov;
   ```
   Execute `$ npm run dev` to create the tables
```

Comment:
`seed.sql` is still using stackoverflowv2

Reply:
Reworked the sql scripts in 35b0f6b